app.py

```diff
@@ -95,23 +133,23 @@ async def create_port_webhook():
         },
         "integrationType": "custom",
     }
-
-    try:
-        response = await client.post(
-            f"{PORT_API_URL}/webhooks",
-            json=webhook_data,
-            headers=port_headers,
-        )
-        response.raise_for_status()
+    response = await send_port_request(
+        "POST",
+        f"webhooks",
```

Comment:
remove the f"" since it's not doing anything

Reply:
Done